centreon/www/include/configuration/configKnowledge/functions.php

```diff
@@ -11,6 +11,32 @@
  *
  */
 
+/**
+ * This returns an anonymous class to manage alternate CSS class for table lines TR.
+ */
+function getLineTemplate(string $class1, string $class2): object
+{
+    return new class($class1, $class2)
+    {
+        private int $number = 0;
+
+        public function __construct(private string $class1, private string $class2)
+        {
+        }
+
+        public function get(): string
+        {
+            return ($this->number++ % 2) ? $this->class2 : $this->class1;
+        }
```

Comment:
Can you add a comment to explain the incrementation and modulo 2

Reply:
The meaning of `$i++` or `++$i` should be explained ?
I wanted to oneline, but I can split into 2 lines if it is not clear enough with the last change and the renaming of the two properties.